win_build/wrapper_vs2019.vcxproj

```diff
@@ -90,10 +90,7 @@
     <CodeAnalysisRules Condition="'$(Configuration)|$(Platform)'=='Release|ARM64'" />
     <CodeAnalysisRuleAssemblies Condition="'$(Configuration)|$(Platform)'=='Release|x64'" />
     <CodeAnalysisRuleAssemblies Condition="'$(Configuration)|$(Platform)'=='Release|ARM64'" />
-    <TargetName Condition="'$(Configuration)|$(Platform)'=='Release|x64'">wrapper_26016_windows_x86_64</TargetName>
-    <TargetName Condition="'$(Configuration)|$(Platform)'=='Release|ARM64'">wrapper_26016_windows_arm64</TargetName>
-    <TargetName Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">wrapper_26016_windows_x86_64</TargetName>
-    <TargetName Condition="'$(Configuration)|$(Platform)'=='Debug|ARM64'">wrapper_26016_windows_arm64</TargetName>
+    <TargetName>$(ProjectName)_26016_windows_$(Platform)</TargetName>
```

Comment:
This will change architecture name from `x86_64` to `x64`. For compatibility target name architecture should be `x86_64`.

Reply:
The architecture is x64.

Reply:
And should be `x86_64`

Reply:
And should be `x64`